src/test/java/com/vmware/test/i18n/I18nUtilTest.java

```diff
@@ -25,7 +25,7 @@ public class I18nUtilTest {
 	final String testURL = "https://github.com/unicode-cldr/cldr-core/archive/32.0.0.zip";
 	final String[] catesArr = { "dates", "numbers", "plurals", "measurements"};
 
-	@Before
```

Comment:
Why commented?

Reply:
because this is test the  download 32.0.0 cldr data. we don't need to test it